pages/about.html

```diff
@@ -5,4 +5,46 @@
 home: false
 ---
 
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<section id="our-team">
+    <h2>Who leads Code for Chicago?</h2>
+    <div class="leader-card">
+        <img alt="picture of Ryan Koch" src="\images\proxy-img.png"><!--Replace proxy image when actual picture becomes avaiable-->
```

Comment:
"picture of [name]" is a little redundant. 
Can you switch to alt="Ryan Koch", etc.. ?